certz/certz.proto

```diff
@@ -223,6 +288,36 @@ message RotateCertificateResponse {
 message FinalizeRequest {
 }
 
+// A request to add one or more SSL profiles.
+message AddProfileRequest {
+  // A list of IDs of SSL profiles to be created.
+  repeated string ssl_profile_ids = 1;
+}
+
+// A response to a request to add one or more SSL profiles.
+message AddProfileResponse {
+}
+
+// A request to remove one or more SSL profiles.
+message DeleteProfileRequest {
```

Comment:
This has the same issue as the [above comment ](https://github.com/openconfig/gnsi/pull/51/files#r1021579217) regarding `AddProfileRequest`
i.e. its not clear how the target should handle an error if one of the ID's fails to be removed successfully. 

Whatever approach is chosen for solving this issue in the case of `AddProfileRequest` should also be used here for consistency

Reply:
I have removed the `repeated` keyword. Please taka a look.